src/components/PresentationContext.tsx

```diff
@@ -23,10 +25,12 @@ export interface PresentationFieldProps {
   errorInTooltip?: true;
   /** Allow the fields to grow to the width of its container. By default, fields will extend up to 550px */
   fullWidth?: boolean;
+  /** Changes bg and hoverBg; Takes priority over `contrast`; Useful when showing many fields w/in a table that require user attention; In no way should be used as a replacement for error/focus state */
+  inputStylePalette?: InputStylePalette;
 }
 
 export type PresentationContextProps = {
-  fieldProps?: PresentationFieldProps;
+  fieldProps?: Omit<PresentationFieldProps, "inputStylePalette">;
```

Comment:
Not something we want set at a context level

Reply:
Makes sense, that it's not a context-level thing, but then wonder if we can find a better spot for declaring it, then `PresentationFieldProps`?

Hm, I guess `PresentationFieldProps` does seem to be "common across text fields and select fields", and it isn't used by other things that aren't getting supported in this PR... :thinking: so sgtm...

Could make your PR comment an in-source comment :shrug:

Reply:
> "common across text fields and select fields",
Yup! Thats why its here. Comment updated